packages/node-fetch/tests/fetch.spec.ts

```diff
@@ -120,6 +121,39 @@ describe('Node Fetch Ponyfill', () => {
         }),
       ).rejects.toThrow('aborted');
     });
+    // TODO: Remove .only on test before merging.
+    // TODO: Remove console.logs before merging.
+    it.only('should respect AbortSignal on a streamed response', async () => {
+      console.log(`should respect AbortSignal on a streamed response (${implementationName})`);
+      expect.assertions(1);
+      const controller = new AbortController();
+      const fetchPromise = fetchPonyfill(baseUrl + `/stream-bytes/${10 * 1024 * 1024 * 1024}`, {
+        signal: controller.signal,
+      });
```

Comment:
This works for calling httpbin. I notice you have another local runner, is that just a proxy to httpbin? If not we may need to update that local server runner to include this endpoint.